### ISSUE TITLE
[Bug]: Multiple attached projects only uses plugin on first opened one

### ISSUE THREAD

**Issue by skylord123** [laravel-idea/plugin#671]
### Bug description

If you open multiple Laravel projects in PHPStorm and attach them to the same window only the first opened project will be working with this plugin.

I would like for it to treat each opened project as it's own laravel project. For someone working on Laravel microservices this is a must have.

### Plugin version

PHPStorm 2022.2.3

### Operating system

Windows

### Steps to reproduce

1) Open multiple projects in PHPStorm and attach to same window
2) Verify that the plugin only is working for first opened project

**Comment by adelf** [comment]
Hello.

Yes, it's a known issue. It will be very hard, almost impossible, to implement. The current architecture of PhpStorm and Laravel Idea just doesn't allow this. Sorry.

**Comment by skylord123** [comment]
> Hello.
> 
> Yes, it's a known issue. It will be very hard, almost impossible, to implement. The current architecture of PhpStorm and Laravel Idea just doesn't allow this. Sorry.

Yeah that is something I was worried about. Dang. Guess I will have to open an issue with JetBrains then.

The same issue is with the detected PHP version. It will only detect the php version of the first opened app. Good times.

Thanks for the reply!
